### DISCUSSION TITLE
tooltip cannot be found in patrol but in flutter integration its ok.

### DISCUSSION THREAD

**Discussion by ccfiel** [leancodepl/patrol#1241] [Q&A]
This is the widget "FloatingActionButton"
![SCR-20230429-jhhq](https://user-images.githubusercontent.com/1788360/235278738-3f4d271e-fbab-4ac3-9692-d62043acea64.png)

in normal flutter integration test and this works. It can find the widget.
```
      final Finder fab = find.byTooltip('Increment');
      await tester.tap(fab);
```

but in patrol it cannot find. This is the error "Finder "zero widgets with
text "Increment" (ignoring offstage widgets)" did not find any
visible"

```
      await $('Increment').tap();
```

Any idea why?

**Comment by bartekpacia** [comment]
Hi,

Patrol's custom finder (the dollar `$`) currently cannot find tooltips.

Thanks for reporting this, we'll consider adding this feature in a future release.

@jBorkowska, @mateuszwojtczak wdyt?

**Reply by lehmannk** [reply]
any updated plans on this?
